vite.config.js

```diff
@@ -18,8 +18,15 @@ module.exports = defineConfig({
           root,
           'pages',
           'developers-guide',
-          'index.html',
+          'index.html'
         ),
+        collectionDetails: resolve(
+          root,
+          'pages',
+          'collection-details',
+          'index.html'
+        ),
+        about: resolve(root, 'pages', 'about', 'index.html'),
```

Comment:
po co ta zmiana?

Reply:
odpal: npm run build

Reply:
Przepraszam, błędnie założyłem że jak działa na 'npm run dev' to jest dobrze i nie sprawdziłem czy się buduje.